backend/internal/test/integration/xds/cmd/envoyconfiggen/main.go

```diff
@@ -13,7 +13,7 @@ name: envoy.fault
 typed_config:
   '@type': type.googleapis.com/envoy.extensions.filters.http.fault.v3.HTTPFault
   abort:
-    http_status: 400
+    http_status: 503
```

Comment:
Doesn't this make it hard to tell the difference between an injected fault and a 503 UC?